src/clients/HubPoolClient.ts

```diff
@@ -70,17 +70,19 @@ type L1TokensToDestinationTokens = {
 };
 
 // Temporary type for v2 -> v3 transition. @todo: Remove.
-export type V2PartialDepositWithBlock = Pick<
+type V2PartialDepositWithBlock = Pick<
   V2DepositWithBlock,
-  "originChainId" | "destinationChainId" | "originToken" | "amount" | "quoteTimestamp" | "blockNumber"
```

Comment:
fwiw `blockNumber` was not actually required in this type and was probably there by error.

Reply:
I actually think this is important to get the MockHubPoolClient to be able to mock an LP fee: 262ba0762b87d8801e9d2efa0eef5f62033deb3d

This is required to make the tests easier in the BundleDataClient because it calls `hubPoolClient.computeLpFee` instead of loading the realized lp fee stored in Deposits at SpokePoolClient.update time